apis/pipelines/v1alpha5/conversion.go

```diff
@@ -1,55 +1,87 @@
 package v1alpha5
 
 import (
-	"github.com/sky-uk/kfp-operator/apis"
+	hub "github.com/sky-uk/kfp-operator/apis/pipelines/v1alpha6"
 )
 
-type RunConfigurationConversionRemainder struct {
-	RunConversionRemainder `json:",inline"`
-	Triggers               Triggers `json:"triggers,omitempty"`
-}
-
-func (rcr RunConfigurationConversionRemainder) Empty() bool {
-	return len(rcr.Triggers.Schedules) == 0 && len(rcr.Triggers.OnChange) == 0 && rcr.RunConversionRemainder.Empty()
-}
-
-type RunConversionRemainder struct {
-	Artifacts           []OutputArtifact   `json:"artifacts,omitempty"`
-	ValueFromParameters []RuntimeParameter `json:"valueFromParameters,omitempty"`
-}
-
-func (rcr RunConversionRemainder) Empty() bool {
-	return len(rcr.Artifacts) == 0 && len(rcr.ValueFromParameters) == 0
+func convertRuntimeParametersTo(rtp []RuntimeParameter) []hub.RuntimeParameter {
+	convertValueFromTo := func(v *ValueFrom) *hub.ValueFrom {
+		if v != nil {
+			return &hub.ValueFrom{
+				RunConfigurationRef: hub.RunConfigurationRef{
+					Name:           v.RunConfigurationRef.Name,
+					OutputArtifact: v.RunConfigurationRef.OutputArtifact,
+				},
+			}
+		}
+		return nil
+	}
+	var hubRtp []hub.RuntimeParameter
+	for _, namedValue := range rtp {
+		hubRtp = append(hubRtp, hub.RuntimeParameter{
+			Name:      namedValue.Name,
+			Value:     namedValue.Value,
+			ValueFrom: convertValueFromTo(namedValue.ValueFrom),
+		})
+	}
+	return hubRtp
 }
 
-func (rcr RunConversionRemainder) ConversionAnnotation() string {
-	return GroupVersion.Version + "." + GroupVersion.Group + "/conversions.remainder"
+func convertArtifactsTo(oa []OutputArtifact) []hub.OutputArtifact {
```

Comment:
would prefer usage of `oa` in this function were more explicitly just `outputArtifact`

Reply:
done in 1060cf492503107798674c9aa4752556a45ed919